content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,162 @@
+
+  -
+    image: /static/images/experiments/lockbox/details/sync.png
+    copy: >
+      Manage accounts in Firefox and sync changes automatically to Firefox Lockbox.
+
+  -
+    image: /static/images/experiments/lockbox/details/sort-filter.png
+    copy: >
+      Sort and filter on your accounts to quickly locate the one you need.
+
+contributors:
+  -
+    display_name: 'Sandy Sage'
+    title: 'Sr. Product Manager'
+    avatar: /static/images/experiments/lockbox/avatars/ssage.png
+  -
+    display_name: 'Devin Reams'
+    title: 'Engineering Manager'
+    avatar: /static/images/experiments/lockbox/avatars/dreams.png
+  -
+    display_name: 'Ryan Gaddis'
+    title: 'Sr. Interaction Designer'
+    avatar: /static/images/experiments/lockbox/avatars/rgaddis.png
+  -
+    display_name: 'Matt Miller'
+    title: 'Engineering Architect'
+    avatar: /static/images/experiments/lockbox/avatars/mmiller.png
+  -
+    display_name: 'Sasha Heinen'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/sheinen.png
+  -
+    display_name: 'Jim Porter'
+    title: 'Sr. Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/jporter.png
+  -
+    display_name: 'James Hugman'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/jhugman.png
+  -
+    display_name: 'Leif Oines'
+    title: 'Product Data Scientist'
+    avatar: /static/images/experiments/lockbox/avatars/loines.png
+  -
+    display_name: 'Heather McGaw'
+    title: 'Sr. UX Researcher'
+    avatar: /static/images/experiments/lockbox/avatars/hmcgaw.png
+  -
+    display_name: 'Michelle Heubusch'
+    title: 'Content Strategy Lead'
+    avatar: /static/images/experiments/lockbox/avatars/mheubusch.png
+  -
+    display_name: 'Nick Brandt'
+    title: 'Sr. UX Designer'
+    avatar: /static/images/experiments/lockbox/avatars/nbrandt.png
+  -
+    display_name: 'Peter deHaan'
+    title: 'QA Developer'
+    avatar: /static/images/experiments/lockbox/avatars/pdehaan.png
+  -
+    display_name: 'Joe Gasiorek'
+    title: 'Community Contributor'
+    avatar: /static/images/experiments/lockbox/avatars/jgasiorek.png
+  -
+    display_name: 'Paul Nyondo'
+    title: 'Community Contributor'
+    avatar: /static/images/experiments/lockbox/avatars/pnyondo.png
+
+# contributors_extra: >
+#     This experiment was also made possible by a generous grant from
+#     the Institute for RoboCop Studies.
+# contributors_extra_url: https://robocop.org
+tour_steps:
+  -
+    image: /static/images/experiments/lockbox/tour/tour-1.jpg
+    copy: >
+      <p>Lorem ipsum dolor sit amet.</p>
+
+measurements:
+  - >
+    PUT SOMETHING ELSE HERE.
+
+pre_feedback_image: /static/images/experiments/experiment-name/pre-feedback.jpg
+pre_feedback_copy: >
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quibusdam atque accusamus,
+    suscipit, nam commodi excepturi error modi.</p>
+
+graduation_url: 'http://example.com/graduation-report'
```

Comment:
I think a lot of the following items are placeholders and may need some :eyes: before merging.
Like, we probably don't want `eol_warning` or `incompatible` here.